src/main/resources/assets/dungeons_animations/geo/illager_geo.geo.json

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
Your illager_geo is too different from the one we just added in Dungeons Mobs. Please use that one and rework all names towards that for consistency.